### DISCUSSION TITLE
FlixTrains als Regionalexpress eingestuft?

### DISCUSSION THREAD

**Discussion by archaeosmith** [Traewelling/traewelling#2364] [Questions / Support] [answered]
Habe gerade zwei FlixTrain fahrten eingegeben (FFS-BHBF und zurück), und sie erscheinen in den Statistiken als Regionalexpresse. Dabei sind sie doch Fernverkehrszüge - am besten mit IC/EC einzustufen, auch wenn sie in den Tabellen ihre eigene Zuggattung (FLX) haben.

**Comment by ChillLP** [comment]
Kein Träwelling Problem per-se, da die Daten lediglich von der DB (glaub ich) über die HAFAS API Bereitgestellt werden und diese dort dann als Regionalzug eingestuft wurden.

**Reply by ChillLP** [reply]
korregiert mich falls ich falsch liege

**Reply by archaeosmith** [reply]
Das verstehe ich auch so, nur würde ich mir dann (wenn möglich) einen eingebauten Script wünschen, der bei der Gattung FLX ausnahmsweise die Einstufung nicht von Hafas übernimmt, sondern gezielt IC/EC eingibt.

**Reply by HAMSmh** [reply]
FlixTrains werden von der DB (zutreffenderweise) als D-Zug eingestuft, dies ist laut der Verkehrsmittelauswahl im DB Navigator eine Fernverkehrsgattung, ich hab mich das auch schon bei diversen Nachtzügen dritter (in meinem Fall der SJ-Nachtzug nach Stockholm) gefragt ob da Träwelling in den Kategorien nicht etwas ändern sollte

angaben ohne gewähr xd

**Reply by SoraCent** [reply]
In DB-Nav und bahn.de werden FLX-Züge als Interregio- und Schnellzüge kategorisiert. Im HAFAS sind FLX-Züge als Produkt "regionalExpress" definiert. Interregio Züge werden im HAFAS immer als "regionalExpress" definiert, in Träwelling gibts zwischen Regionalexpress und IC auch keine andere Punkte Stufe weshalb es sinn macht das FLX-Züge welche bei der DB als Interregio angesehen werden dann in Träwelling als RegionalExpress statt IC angenommen werden.

**Comment by d3d9** [comment, marked as answer]
Haben es alle im Blick, dass das regionalExp(ress) aus der db-rest-Response nichts mit Regionalexpress-Zügen, gemeinhin als RE unterwegs, zu tun hat, außer, dass sie ähnlich heißen?

Ich finde es ärgerlich, dass das dort so bezeichnet wird, weil es bei Nutzenden der Daten zu dieser Fehlinterpretation führt.

Beim Abfahrtsmonitor-Filter müsste diese Art einen eigenen Button bekommen (IR/D o.ä.) statt unter Regional mit gefilfert zu werden. Dann könnte man aber auch in Erwägung ziehen, den Fernverkehr-Button aufzuteilen in IC/EC und ICE. Oder regionalExp mit unter Fernverkehr berücksichtigen, zumindest wenn man den Text des teilweise sowieso die volle Breite einnehmenden Buttons mit einer Klammer mit Auflistung versieht. *

Für Statistiken und andere Stellen wo regionalExp eigenständig auftaucht, müsste die verwendete menschenlesbare Bezeichnung nur angepasst werden, also statt "Regionalexpress" eher "IR/D-Zug" oderso.


\* (außerdem könnte man bei einer Überarbeitung der Filter zusätzlich Taxen berücksichtigen die bisher keinen Filter-Button haben, und auch ohne jegliche Filter nicht abgefragt werden)

**Reply by Prerlesuxa** [reply]
Wie bereits von anderen Benutzern erwähnt, werden Interregio Züge einschließlich Flixzüge als RegionalExpress in der db-rest-Response betrachtet. Im Gegensatz dazu werden RegionalExpress Züge als regional eingestuft.
In der db-rest-Response werden alle Produkte der API und von DB angezeigt, bei dieser auflistung ist es offensichtlich, dass es keine zusätzlichen Produkte zwischen RegionalExpress und National (IC, EC) gibt. Folglich sind InterregioExpress und Flixzüge höher als das Produkt Regional und gehören somit zum Produkt Regionalexpress, während RegionalExpress Züge zum Produkt Regional gehören.

**Reply by archaeosmith** [reply]
Verstehe. Verwirrend ist die Beschreibung "Regionalexpress" für D-Züge allemal. Aber naja, solang sie gesondert von sowohl IC/EC als auch Regionalzügen ("Regional") in der Statistik eingetragen sind, ist ja alles gut!
